include/seqan/align/dp_formula_affine.h

```diff
@@ -199,13 +189,9 @@ _internalComputeScore(DPCell_<TScoreValue, AffineGaps> & activeCell,
                       TracebackOn<TracebackConfig_<SingleTrace, TGapsPlacement> >  const &,
                       RecursionDirectionHorizontal const &)
 {
-    if(activeCell._horizontalScore < rightCompare)
-    {
-        activeCell._score = activeCell._horizontalScore = rightCompare;
-        return rightTrace;
-    }
-    activeCell._score = activeCell._horizontalScore;
-    return leftTrace;
+    return (activeCell._horizontalScore < rightCompare) ?
+        (activeCell._horizontalScore = rightCompare, rightTrace) :
+        (leftTrace);
```

Comment:
Well I followed the style guide suggesting to wrap the line after the operator.
But I agree that this looks more readable.